runtime-tests/index.ts

```diff
@@ -0,0 +1,39 @@
+const config = {
+	chain: 'polkadot',
+};
+const argv = process.argv.slice(0, 2);
+
+/**
+ * We return empty string's as an indication of a false value. JS recognizes "" as falsey.
+ * NodeJS.Process.argv.reduce expects the return value of the accumulator to
+ * be of type string, this is a way of satisfying the compiler, but also
+ * returning the correct behavior.
+ */
+process.argv.reduce((cmd, arg) => {
```

Comment:
```suggestion
process.argv.slice(2).reduce((cmd, arg) => {
```
Shouldn't we not be checking the first two elements of argv since they are the program?

Reply:
Yea that totally makes sense, I just completely changed this whole file, and made it way simpler with an arg parser.